File: back_end/src/main/java/com/lukeshay/restapi/security/AuthBody.java
package com.lukeshay.restapi.security;

import com.google.gson.annotations.Expose;
import com.lukeshay.restapi.session.Session;
import com.lukeshay.restapi.user.User;
import com.lukeshay.restapi.utils.Models;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class AuthBody {
  @Expose private User user;
  @Expose private Session session;

  @Override
  public String toString() {
    return Models.toString(this);
  }
}

File: back_end/src/main/java/com/lukeshay/restapi/config/security/AuthBody.kt
package com.lukeshay.restapi.config.security

import com.google.gson.annotations.Expose
import com.lukeshay.restapi.session.Session
import com.lukeshay.restapi.user.User
import com.lukeshay.restapi.utils.Models;

class AuthBody(@Expose val Authorization: String, @Expose val user: User, @Expose val session: Session) {

    /**
     * Returns a string representation of the object.
     */
    override fun toString(): String {
        return Models.toString(this)
    }
}